tools/wpt/browser.py

```diff
@@ -274,7 +275,7 @@ def find_webdriver(self, channel=None):
 
     def get_version_and_channel(self, binary):
         version_string = call(binary, "--version").strip()
-        m = re.match(r"Mozilla Firefox (\d+\.\d+(?:\.\d+)?)(a|b)?", version_string)
+        m = re.match(r"Mozilla Firefox (\d+\.\d+(?:\.\d+)?)(a|b)?", ensure_str(version_string))
```

Comment:
Does this actually have to be bytes in Py2, or could we use the same type in both cases?

Reply:
updated patches

Reply:
This is addressed  in https://github.com/web-platform-tests/wpt/pull/21656